Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/API.hs

```diff
@@ -133,9 +141,18 @@ makeBookingAPIEntity booking activeRide allRides fareBreakups = do
               ..
             }
 
-buildBookingAPIEntity :: EsqDBReplicaFlow m r => Booking -> m BookingAPIEntity
+buildBookingAPIEntity :: (CacheFlow m r, EsqDBFlow m r) => EsqDBReplicaFlow m r => Booking -> m BookingAPIEntity
 buildBookingAPIEntity booking = do
   mbRide <- runInReplica $ QRide.findActiveByRBId booking.id
   rideList <- runInReplica $ QRide.findAllByRBId booking.id
   fareBreakups <- runInReplica $ QFareBreakup.findAllByBookingId booking.id
-  return $ makeBookingAPIEntity booking mbRide rideList fareBreakups
+  exophone <- findRandomExophone booking.merchantId
```

Comment:
Every time when we fetch `bookingList` we will receive different random exophone. Is it correct logic?